src/components/Dashboard/SettingsMenu.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, Fragment } from 'react';
+import { makeStyles } from '@material-ui/styles';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import IconButton from '@material-ui/core/IconButton';
+import Settings from '@material-ui/icons/Settings';
```

Comment:
For icons, let's use `mdi-react` which has all material icons + community icons.

```jsx
import SettingsOutlineIcon from 'mdi-react/SettingsOutlineIcon';
```